config/config.go

```diff
@@ -60,9 +62,8 @@ func Init() error {
 		configDir = amforaAppData
 	} else {
 		// Unix / POSIX system
-		xdg_config, ok := os.LookupEnv("XDG_CONFIG_HOME")
-		if ok && strings.TrimSpace(xdg_config) != "" {
-			configDir = filepath.Join(xdg_config, "amfora")
+		if basedir.ConfigHome != "" {
+			configDir = filepath.Join(basedir.ConfigHome, "amfora")
 		} else {
 			// Default to ~/.config/amfora
 			configDir = filepath.Join(home, ".config", "amfora")
```

Comment:
```suggestion
		if basedir.ConfigHome == "" {
			// Default to ~/.config/amfora
			configDir = filepath.Join(home, ".config", "amfora")
		} else {
			configDir = filepath.Join(basedir.ConfigHome, "amfora")
```

Switch to a positive if statement.